### PR TITLE
Add new ./run script for building

### PR DESCRIPTION
**Pull request by nottrobin**
Inline with the latest methods for local development recently added to
www.ubuntu.com project, I am adding a similar ./run script to this project.

You can now run any of:

- `./run build`
- `./run watch`
- `./run test`
- `./run clean`

Each task will be run within Docker containers, so you don't need yarn or
npm locally, and you don't need to worry about whether you're using the
correct version.

NPM dependencies will also be automatically managed so you don't need to
ever run `npm install` or `yarn install`.

QA
---

Start with `./run --help`. Then try running the available commands. Check they work as expected.

Also, read [the new README](https://github.com/nottrobin/vanilla-docs-theme/blob/run-script/README.md).

### CODE REVIEW

File: scss/_patterns_navigation.scss [hunk 1]
```diff
@@ -11,8 +11,8 @@
     border-bottom: 1px solid $color-mid-light;
     color: $color-dark;
     line-height: 32px;
-    padding: 7.5px 0;
     margin-bottom: 0;
+    padding: 7.5px 0;
```
Comment (nottrobin) [inline scss/_patterns_navigation.scss]: :heart:

**Comment by barrymcgee** [comment]
All looks great - only issue I see is when running `./run test`

screen shot 2017-03-21 at 15 35 37

**Comment by nottrobin** [comment]
@barrymcgee good catch. I'll fix that now.

**Comment by nottrobin** [comment]
@barrymcgee I've fixed that up

**Comment by barrymcgee** [comment]
Looks good now 👌🏻

Can you also use this setup to specify the project uses the local version of a module by passing a flag? If so, can you document that in the README please?

**Comment by nottrobin** [comment]
@barrymcgee done. Merging now.

**Pull request merged**